### DISCUSSION TITLE
Downgrading to different type of flyby?

### DISCUSSION THREAD

**Discussion by peyre** [raceintospace/raceintospace#608] [Ideas]
I'm in a game now where I scheduled a Venus flyby but then the computer beat me to it.  Now I have a Venus flyby in the VIB which is useless to me.  That got me thinking: what if you could "downgrade" it to a Mercury or even a lunar flyby?  That would be awesome game-wise, but I don't know if it's realistic at all.  Of course we're talking about simple flybys, so not nearly as complex as a landing - but I don't know, does something like that take too much planning to turn on a time like that?

**Comment by rnyoakum** [comment]
I'm not sure what you mean about it being useless, because you still get 4 prestige points for the second-place achievement. It may not be as cost-efficient, but it isn't _nothing_.

**Reply by peyre** [reply]
That's a good point, now you mention it.
